src/App.tsx

```diff
@@ -1,10 +1,45 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
You dont need import React here. For example u can write 'FC' instead of 'React.FC'